src/main/java/edu/hm/hafner/analysis/registry/VeracodePipelineScannerDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.VeracodePipelineScannerParser;
+
+import static j2html.TagCreator.*;
+
+/**
+ * A descriptor for Veracode Pipeline Scanner.
+ *
+ * @author Juri Duval
+ */
+class VeracodePipelineScannerDescriptor extends ParserDescriptor {
```

Comment:
Please use Camel Case: 
```suggestion
class VeraCodePipelineScannerDescriptor extends ParserDescriptor {
```

Reply:
I believe Veracode is not written as VeraCode. See https://en.wikipedia.org/wiki/Veracode.
Would you still expect it to be camel cased?

Reply:
If you don't mind, then yes. In Java everything is using CamelCase even though the original name does not. But if you want to stick with the company name, I am fine.

Reply:
Sure, will update this now.

Reply:
Fixed.